.github/workflows/deploy.yml

```diff
@@ -0,0 +1,102 @@
+name: Publish and Deploy Docker image
+
+on:
+  workflow_run:
+    workflows: ["Test and scan Docker image"]
+    branches: [main]
+    types:
+      - completed
+  workflow_dispatch:
+
+env:
+  REGISTRY: ghcr.io
+  IMAGE_NAME: ghcr.io/${{ github.repository }}
+  AZURE_WEBAPP_NAME: catalogue-test
+  DOCKER_METADATA_OUTPUT_TAGS:
+
+jobs:
+  ort:
+    if: ${{ github.event.workflow_run.conclusion == 'success' }}
+    strategy:
+      fail-fast: false
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: oss-review-toolkit/ort-ci-github-action@7f23c1f8d169dad430e41df223d3b8409c7a156e
+        with:
+          report-formats: CycloneDx,SpdxDocument,WebApp,PlainTextTemplate,Opossum
+          run: "cache-dependencies,cache-scan-results,labels,analyzer,evaluator,advisor,reporter,upload-results"
+
+  scan-docker-image:
+    runs-on: ubuntu-latest
+    needs: ort
+    permissions:
+      contents: read
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Extract metadata (tags, labels) for Docker
+        id: meta
+        uses: docker/metadata-action@9ec57ed1fcdbf14dcef7dfbe97b2010124a938b7
+        with:
+          images: ${{ env.IMAGE_NAME }}
+
+      - name: Build and push Docker image
```

Comment:
its a bit unclear here, are you building and pushing it somewhere to do the scan? Because you also have similar step on line 85

Reply:
True, the name of this step is misleading. I'll fix it.

Reply:
Fixed.